File: msgraph-mail/dotnet/Users/MailFolders/Messages/AttachmentsResponse.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
namespace Graphdotnetv4.Users.MailFolders.Messages {
    public class AttachmentsResponse {
        public List<Attachment> Value { get; set; }
        public string NextLink { get; set; }
    }
}

File: msgraph-mail/swiftconsoleapp/Sources/swiftconsoleapp/Users/Item/MailFolders/Item/Messages/Item/Attachments/AttachmentsResponse.swift
extension Swiftconsoleapp.Users.Item.MailFolders.Item.Messages.Item.Attachments {
    public class AttachmentsResponse : IAdditionalDataHolder {
        public var additionalData: IDictionary<string, object>
        public var nextLink: String
        public var value: [Attachment]
    }
}
